tonic-examples/proto/blobservice/blobservice.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright 2015 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This is also not accurate, I think. It's Apache OR MIT dual-licensed.

Reply:
As an addendum, the right thing to do if license headers are desired is to use SPDX format ones, otherwise the LICENSE in the repo works just fine.